apps/app/dashboards/index.tsx

```diff
@@ -73,6 +75,8 @@ const DashboardIndex: FunctionComponent<DashboardIndexProps> = ({
     for (const [category, dbs] of Object.entries(dashboards)) {
       const dashboards = dbs
         .filter(d => {
+          if (d.name === "covid-vaccination") Object.assign(d, { fillColor: AKSARA_COLOR.GREEN });
+          else if (d.agency === "aadk") Object.assign(d, { fillColor: AKSARA_COLOR.DANGER });
```

Comment:
Can get messy really quick if you have more conditional coloring like this. Something to think about later

Reply:
the other method would probably to add a prop on data side?

Reply:
Not really (reason: keep styling config on FE). Take a step back and think it through. For now, i'll accept it, but this is for you to explore in your free time.